components/footer/Footer.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  FAUCETS,
+  FooterLinkInfo,
+  LEGAL,
+  NETWORKS,
+  RESOURCES,
+  SDKs,
+  SOLUTIONS,
+} from "./footerLinks";
+import { SOCIALS } from "./socialLinks";
+import {
+  Box,
+  ButtonGroup,
+  Container,
+  Flex,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react";
+import { Logo } from "components/logo";
+import { PRODUCTS } from "components/product-pages/common/nav/DesktopMenu";
+import {
+  Heading,
+  LinkButton,
+  TrackedIconButton,
+  TrackedLink,
+} from "tw-components";
+
+type FooterLinkGroupProps = {
```

Comment:
we've been using `interface` for most props - not a blocker though

Reply:
Changed `type` to `interface` in https://github.com/thirdweb-dev/dashboard/pull/716/commits/a32c1fcf2b8a4bc6815c9f3ca73f7d30890e14c7